### ISSUE TITLE
Angular version of Intro to Storybook error on "npm install -D @storybook/addon-storyshots"

### ISSUE THREAD

**Issue by norcino** [chromaui/learnstorybook.com#498]
I followed all steps from [here](https://storybook.js.org/tutorials/intro-to-storybook/angular/en/simple-component/).

When I get to [Snapshot testing](https://storybook.js.org/tutorials/intro-to-storybook/angular/en/simple-component/) and I try to run "npm install -D @storybook/addon-storyshots", I get the following.

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! Found: react@16.14.0
> npm ERR! node_modules/react
> npm ERR!   react@"16.14.0" from @storybook/angular@6.2.8
> npm ERR!   node_modules/@storybook/angular
> npm ERR!     dev @storybook/angular@"^6.2.8" from the root project
> npm ERR!     peerOptional @storybook/angular@"*" from @storybook/addon-storyshots@6.2.9
> npm ERR!     node_modules/@storybook/addon-storyshots
> npm ERR!       dev @storybook/addon-storyshots@"*" from the root project
> npm ERR!   peerOptional react@"^16.8.0 || ^17.0.0" from @storybook/addon-storyshots@6.2.9
> npm ERR!   node_modules/@storybook/addon-storyshots
> npm ERR!     dev @storybook/addon-storyshots@"*" from the root project
> npm ERR!   5 more (@storybook/react, @storybook/addons, ...)
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"17.0.2" from react-test-renderer@17.0.2
> npm ERR! node_modules/@storybook/addon-storyshots/node_modules/react-test-renderer
> npm ERR!   react-test-renderer@"^16.8.0 || ^17.0.0" from @storybook/addon-storyshots@6.2.9
> npm ERR!   node_modules/@storybook/addon-storyshots
> npm ERR!     dev @storybook/addon-storyshots@"*" from the root project
> npm ERR!
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> npm ERR!
> npm ERR! See C:\Users\manuel.salvatori\AppData\Local\npm-cache\eresolve-report.txt for a full report.
> 
> npm ERR! A complete log of this run can be found in:

I tried to look for a solution myself but with no success, I think that there is some wrong dependency in the example code I was invited to download from [here](https://github.com/chromaui/intro-storybook-angular-template).
From there you can see dependencies over react (package-lock.json), maybe is desired, I don't know.
I also tried to follow the indications from https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core, installing the dependency _jest-present-angular_ but also this doesn't help.

**Comment by jonniebigodes** [comment]
@norcino that's a bit strange. But thank you for bringing this to our attention. I'm going to take a look at the issue and see what might be wrong and I'll let you know. 

Stay safe

**Comment by jonniebigodes** [comment]
@norcino I took a look at your issue followed up the tutorial up to that section. And it checks out.

Here's what I did:
- Cloned the repo from the get started section.
- Followed the Simple component to setup the component/stories.
- Installed the addon and set it up.
- Ran `npm run test` and it yielded the following result:
<img width="1424" alt="snapshot-addon-result" src="https://user-images.githubusercontent.com/22988955/120835493-4cbc3d00-c55c-11eb-9458-b89dcabf2f4f.png">

Here's what I would like for you to do so that we can triage it better:
- Create a new project following the exact steps you took up until where the issue you got started. Probably make a single commit for each step.
- Hoist the code to a GitHub repo and share it.

We'll take a look at it and if it's an actual issue with the addon we can escalate this to the Storybook repo and check with the rest of the maintainers for help with a fix. Sounds good?

Stay safe
